python/gtsam/preamble/basis.h

```diff
@@ -0,0 +1,12 @@
+/* Please refer to:
```

Comment:
@varunagrawal I added this function for the new multiple-interface-files - can you check if I missed anything?